Makefile

```diff
@@ -589,6 +596,19 @@ endif
 		| tee $(TEST_LOG_DIR)/chaos.json \
 		| ${RENDER_TESTS}
 
+# test-go-diff runs go unit tests for flaky tests detector on CI. It does not require packages. -run value and folders are passed using ADDFLAGS.
+.PHONY: test-go-diff
+test-go-diff: ensure-webassets bpf-bytecode rdpclient $(TEST_LOG_DIR) $(RENDER_TESTS)
```

Comment:
Is `rdpclient` necessary here? You can probably remove it.

Reply:
Frankly, I have no idea That's how it is implemented in test-unit target.

Reply:
It won't hurt anything as-is, so we can leave it.